app/controllers/recipe_controller.rb

```diff
@@ -1,2 +1,37 @@
 class RecipeController < ApplicationController
+  def new
+    @recipe = Recipe.new
+    @ingredients = Ingredient.all
+  end
+
+  def edit
+    @recipe = Recipe.find(params[:id])
+    @ingredients = Ingredient.all
```

Comment:
Same as above, do we want to set the instance variable to `ALL` Ingredients in the db or just the ingredients associated with the recipe?